api/projects/urls.py

```diff
@@ -35,6 +36,11 @@
     DataDogConfigurationViewSet,
     basename="integrations-datadog",
 )
+projects_router.register(
```

Comment:
I don't think this should be here anymore, right?

Reply:
Yep!